packages/zcli-themes/src/lib/getBrandId.ts

```diff
@@ -0,0 +1,20 @@
+import type { Brand } from '../types'
+import { request } from '@zendesk/zcli-core'
+import * as inquirer from 'inquirer'
+
+export default async function getBrandId (): Promise<string> {
+  const { data: { brands } } = await request.requestAPI('/api/v2/brands.json')
```

Comment:
Should we handle in case this call fails?

Reply:
Good point! I added error handling and an accompanying tests 👍🏼